resources/todomvc/architecture-examples/lit/dist/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+	<head>
```

Comment:
Can we use spaces instead of tabs like all other files?

Reply:
+1, done